app/views/catalog/_index_header_hierarchy_default.html.erb

```diff
@@ -8,11 +8,13 @@
     <% end %>
     <% counter = document_counter_with_offset(document_counter) %>
     <%= link_to_document document, document_show_link_field(document), counter: counter %>
+    <% if document.children? %>
+      <span class="badge badge-pill badge-secondary al-number-of-children-badge"><%= document.number_of_children %></span>
```

Comment:
Maybe this is a question for @jvine or @ggeisler, but for accessibility reasons, do we need to provide additional context for this number, akin to what we were doing before (1 child, 15 children, etc)?

Reply:
Probably! I'm not sure what the best way to do that is, exactly.

Reply:
My understanding from @ggeisler and @jvine was that not including "children" specifically was part of the design decision. We do perhaps need context, but I'll let them mull over how to specify it.

Reply:
I think @cbeer has a good point. We do intentionally want the visible badge to simply show the number of children, but for a screenreader user, hearing the number alone might not be enough context.

I believe we could add `<span class="sr-only">children</span>` after the number but within the badge span so nothing visibly changes in the UI, but screenreaders would say "3 children":

```
<span class="badge badge-pill badge-secondary al-number-of-children-badge"><%= document.number_of_children %><span class="sr-only">children</span></span>
```
(though I think we also want to use the pluralize method for the "children" text in case there is only one)

We probably should get confirmation from @jvine that this makes sense before we go with it, since she is more of an a11y expert than I am.

Reply:
Was trying to resist this, but Chris is right, we should provide the context. We explicitly removed the word "children" from the UI because of usability feedback. Embedding the same word in "invisible" content will make it less likely that implementers will know it's in there creating confusion. 

Suggestion 1: be explicit about "3 folders" for sub-containers and "3 items" for files. a) do we know? and b) can a parent contain both?

Suggestion 2: "components". Feels like it might be jargon-y because we use it internally, but my independent "what word should I use" consultant came up with it on their own, so maybe?

Reply:
Good point about the term "children". I was just focused on technical implementation and forgot the term "children" was problematic.

Reply:
WFM; it seems like there's enough open questions around suggestion 1 that it could spin off to a new ticket if it's something we want to pursue.